### PR TITLE
Updated deprecated links in desktop-app.rst

### PR DESCRIPTION
**Pull request by Rutam21**
#### Summary

Updated deprecated links in desktop-app.rst

#### Ticket Link

NA

### CODE REVIEW

File: source/deploy/desktop-app.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ Desktop App Deployment Guide
 
 Mattermost desktop applications are available for Windows, macOS, and Linux operating systems.
 
-You can `download the apps directly from our download page <https://mattermost.com/download/#mattermostApps>`__ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`__ for help during setup and for troubleshooting tips.
+You can `download the apps directly from our download page <https://mattermost.com/download/#>`__ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`__ for help during setup and for troubleshooting tips.
```
Comment (justinegeffen) [inline source/deploy/desktop-app.rst:6]: Curious as to why the "mattermostApps" part was removed, as it appears to land in the correct section of the correct page.

File: source/deploy/desktop-app.rst [hunk 2]
```diff
@@ -76,9 +76,9 @@ You can customize and distribute your own Mattermost Desktop application by conf
 Windows App: Pre-Configuration and Silent Deployment
 -----------------------------------------------------
 
-You can distribute the official Windows Desktop App silently to end users, pre-configured with the server URL. You can also set all the `app settings <https://docs.mattermost.com/help/apps/desktop-guide.html#app-options>`__ except for the **Start app on login** option.
+You can distribute the official Windows Desktop App silently to end users, pre-configured with the server URL. You can also set all the `app settings <https://docs.mattermost.com/messaging/managing-desktop-app-options.html>`__ except for the **Start app on login** option.
 
-1. Download the latest Windows installer from the `Mattermost download page <https://mattermost.com/download/#mattermostApps>`__.
+1. Download the latest Windows installer from the `Mattermost download page <https://mattermost.com/download/#>`__.
```
Comment (justinegeffen) [inline source/deploy/desktop-app.rst:81]: Curious as to why the "mattermostApps" part was removed, as it appears to land in the correct section of the correct page.

File: source/deploy/desktop-app.rst (hunk 1 above)
Comment (Rutam21) [reply source/deploy/desktop-app.rst:6]: Well, when we hit that URL it resolves back to # at the end. And if you try again with this current URL containing #, then it still points to the same section of the page. However, if we use this URL for the first time, it just opens up the page. So, probably we can revert this. Thanks for pointing it out.

File: source/deploy/desktop-app.rst (hunk 2 above)
Comment (Rutam21) [reply source/deploy/desktop-app.rst:81]: I would reply the same as above. Reverting this as well. Thanks.

**Comment by Rutam21** [comment]
@justinegeffen Necessary changes are done. Please re-review.

**Review by justinegeffen — APPROVED** [review]
Thank you! :)

**Pull request merged**